packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx

```diff
@@ -59,7 +61,15 @@ const FeatureLine = ({ feature, enabledFeatures }: FeatureLineProps) => {
                 buttonTitle={<Translation id="TR_LEARN_MORE" />}
             />
             <ActionColumn>
-                <Checkbox isChecked={checked} onClick={onChangeFeature} />
+                {config.goToRoute ? (
+                    <Link to={config.goToRoute}>
```

Comment:
:thinking:  Curiously, we never used `<Link to` so far, even though it's the simplest way to work with react-router.
Instead, we use own function [goto](https://github.com/trezor/trezor-suite/blob/3f257bcad4e2620c6c33af7450a2cc640e4dd940/packages/suite/src/actions/suite/routerActions.ts#L99), which has more complex logic.
But this seems sufficient to me, works as expected.

Reply:
Please use it this way (without Link):

```

const handleClick = () => {
  goTo(config.routeName)
}

...

<Button variant="tertiary" onClick={handleClick}>
  <Translation id="TR_GO_TO_EXP_FEATURE" />
</Button
```

Reply:
Solved in 919e03efc1165779a51769ce3298867753cfe1ba